allauth/account/templatetags/account_tags.py

```diff
@@ -3,4 +3,4 @@
 warnings.warn("{% load account_tags %} is deprecated, use {% load account %}",
               DeprecationWarning)
 
-from account import *  # noqa
+from .account import *  # noqa
```

Comment:
hmm strange... these `_tags` are not imported explicitly anywhere, but Django still tries to import them...